components/database/berkeleydb/berkeleydb.p5m

```diff
@@ -21,37 +21,22 @@
 
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2018, Michal Nowak
 #
 
-<transform file path=usr.*/man/.+ -> default mangler.man.stability committed>
 <transform file path=usr/share/doc/bdb/(.*) -> set action.hash usr/docs/%<\1> >
-set name=pkg.fmri \
-    value=pkg:/database/berkeleydb-5@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
 set name=pkg.summary value="Oracle Berkeley DB"
 set name=pkg.description \
     value="Berkeley DB is a computer software library that provides a high-performance embedded database for key/value data."
-set name=info.classification \
-    value=org.opensolaris.category.2008:System/Databases
+set name=info.classification value=org.opensolaris.category.2008:System/Databases
 set name=info.keyword value=bdb
-set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
-set name=org.opensolaris.arc-caseid value=PSARC/2010/300
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file usr/bin/db_archive path=usr/bin/$(MACH32)/db_archive
```

Comment:
Do we ever need 32-bit version of the tools? If we don't, just deliver 64-bit versions. If we do, ship both and restore isaexec usage

Reply:
How do I know we need 32-bit version of the tools? There are certainly packages depending on BDB, not sure on what exactly. Restored `isaexec`.